pango-view/viewer-render.c

```diff
@@ -714,6 +738,10 @@ parse_options (int argc, char *argv[])
      "Interpret text as Pango markup",					NULL},
     {"output",		'o', 0, G_OPTION_ARG_STRING,			&opt_output,
      "Save rendered image to output file",			      "file"},
+    {"output-file", 0, 0, G_OPTION_ARG_STRING,            &opt_output_file,
+     "Set output file-name (default: stdout)",          "filename"},
```

Comment:
This is option is redundant , since `--output` already exists. Also writing to `stdout` if not output file is specified is change in behavior, better write to `stdout` only if filename is `-`.

Reply:
Here's some code which existed previously, before I made any changes.
```
if ((opt_text && argc != 1) || (!opt_text && argc != 2))
```
How am I meant to keep the same behaviour and also add this new functionality?

Reply:
Never mind... I'm sleep deprived and was forgetting count of `argc` always starts at 1. This is now fixed to your liking.